Assets/Scripts/ShipController.cs

```diff
@@ -228,41 +228,28 @@ internal void Update()
     public void giveGold(float amount) {
 		
 		Gold += amount;
-	}
-
-    public float getGold() {
-		
-		return Gold;
+        GameObject.Find("Gold").GetComponent<ResourcesDisplay>().ChangeAmount("G",Gold);
```

Comment:
If you already had to find the object named gold, why do you have to tell it to have the letter G? Shouldn't each resource know its letter?